src/views/logs.js

```diff
@@ -55,13 +73,17 @@ function updateSummary() {
     updateView(summaryData, 'glimpse-logs-popup-summary-value');
 }
 function updateView(data, target) {
-    // TODO: this needs to be update to use icons
-    let content = data.error + ' / '
-        + data.warn + ' / '
-        + data.info;
+    const content = `
+        <span${iconSectionHasValue(data.error)}>${icons.error} ${data.error}</span>
+        <span${iconSectionHasValue(data.warn)}>${icons.warn} ${data.warn}</span>
+        <span${iconSectionHasValue(data.info)}>${icons.info} ${data.info}</span>
+    `;
 
     document.getElementById(target).innerHTML = content;
 }
+function iconSectionHasValue(total) {
```

Comment:
Nit: rename to `iconSectionHasValueClass` or something like that... at first glance I expect a boolean from this function